mrmustard/physics/ansatze.py

```diff
@@ -141,7 +141,7 @@ class PolyExpBase(Ansatz):
     def __init__(self, mat: Batch[Matrix], vec: Batch[Vector], array: Batch[Tensor]):
         self.mat = math.atleast_3d(math.astensor(mat))
         self.vec = math.atleast_2d(math.astensor(vec))
-        self.array = math.atleast_1d(math.astensor(array))
+        self.array = math.atleast_2d(math.astensor(array))
```

Comment:
Realizing now that `math.astensor` is unnecessary here (as well as in 142 & 143) as its called within `math.atleast_2d`

Reply:
That shaved off about 3 secs when running `make test`. These small unnecessary calls adds up.